dts/bindings/sensor/ams,tsl2561.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023, Gustavo Silva
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    OSRAM ams TSL2561 light sensor.
+
+compatible: "ams,tsl2561"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  integration-time:
+    type: int
+    default: 402
```

Comment:
Please explain why you chose this default in the description. Is it a register reset value?

Reply:
Yes, it is the register's reset value. The description was updated to include this information.